src/test/features/claim/routes/pay-by-account.ts

```diff
@@ -75,7 +75,11 @@ describe('Claim : Pay by Fee Account page', () => {
 
     it('should return 500 and render error page when form is valid and cannot save draft', async () => {
```

Comment:
@timja-kainos please add route test for failure in retrieving payment reference.

Reply:
done